server/commands.lua

```diff
@@ -337,6 +337,21 @@ lib.addCommand('me', {
     playerState:set('me', nil, true)
 end)
 
+-- Metadata command
+
+lib.addCommand('setmetadata', {help = 'Set Player Metadata (God Only)', restricted = 'god'}, function(source, args)
+    local Player = exports.qbx_core:GetPlayer(source)
+    if args[1] then
+        if args[1] == "trucker" then
+            if args[2] then
+                local newrep = Player.PlayerData.metadata["jobrep"]
+                newrep.trucker = tonumber(args[2])
+                Player.Functions.SetMetaData("jobrep", newrep)
+            end
+        end
+    end
+end)
```

Comment:
The body doesn't seem to match the name. I read the name and thought this was a general way for admins to set metadata on a player (could be dangerous as admins could easily break contracts by saving data in the wrong format), but then I read the body and the command actually just seems to set a very specific metadata value "jobrep.trucker"

I'm not sure why this would belong in core? It seems very specific to a trucker resource.

Reply:
I will keep it open to all metadata not only trucker